### DISCUSSION TITLE
My RosePine theme and config.

### DISCUSSION THREAD

**Discussion by ItsLemmy** [abenz1267/walker#143] [Show and tell]
Just wanted to share my RosePine inspired config and theme that I daily drive.
Vertical layout, with easily interchangeable colors.

Thanks @abenz1267 for a great projet.   Feel free to add to the repo if you'd like.

[walker-rosepine.tar.gz](https://github.com/user-attachments/files/17599632/walker-rosepine.tar.gz)

**Comment by abenz1267** [comment]
Thanks for sharing. I actually have something in mind for sharing themes...

**Comment by abenz1267** [comment]
Looks great
